src/test/java/hudson/plugins/emailext/plugins/recipients/FirstFailingBuildSuspectsRecipientProviderTest.java

```diff
@@ -117,9 +121,9 @@ public void testAddRecipients() throws Exception {
          * Committers {U,V}.
          * Failed.
          */
-        final FreeStyleBuild build5 = PowerMockito.mock(FreeStyleBuild.class);
-        PowerMockito.when(build5.getPreviousCompletedBuild()).thenReturn(build4);
-        PowerMockito.when(build5.getResult()).thenReturn(Result.FAILURE);
+        final FreeStyleBuild build5 = PowerMockito.spy(new FreeStyleBuild(p));
+        PowerMockito.doReturn(Result.FAILURE).when(build5).getResult();
+        PowerMockito.doReturn(build4).when(build5).getPreviousCompletedBuild();
```

Comment:
This is one reason I loathe mocking tests—they have to be rewritten even if you make _compatible_ changes.